indexer/ftypes_matcher.cpp

```diff
@@ -393,24 +394,32 @@ string DebugPrint(HighwayClass const cls)
   {
     case HighwayClass::Undefined:
       out << "Undefined";
+      break;
     case HighwayClass::Error:
       out << "Error";
+      break;
     case HighwayClass::Trunk:
       out << "Trunk";
+      break;
     case HighwayClass::Primary:
       out << "Primary";
+      break;
     case HighwayClass::Secondary:
       out << "Secondary";
+      break;
     case HighwayClass::Tertiary:
       out << "Tertiary";
+      break;
     case HighwayClass::LivingStreet:
       out << "LivingStreet";
+      break;
     case HighwayClass::Service:
       out << "Service";
+      break;
+    case HighwayClass::Pedestrian: out << "Pedestrian"; break;
     case HighwayClass::Count:
       out << "Count";
-    default:
-      out << "Unknown value of HighwayClass: " << static_cast<int>(cls);
+      break;
```

Comment:
I did not quite catch where you saw the code duplication. There's only one method 
string DebugPrint(HighwayClass const cls) which converts HighwayClass to string for debug purposes. Which advantages do we have if I add the helper?